shanoir-ng-datasets/src/main/java/org/shanoir/ng/examination/service/ExaminationService.java

```diff
@@ -116,6 +116,5 @@
 	 */
 	String addExtraData(Long examinationId, MultipartFile file);
 
-	@PreAuthorize("hasRole('ADMIN') or (hasAnyRole('EXPERT', 'USER') and @datasetSecurityService.hasRightOnExamination(#examinationId, 'CAN_DOWNLOAD'))")
```

Comment:
is this verified by the APIController already?

Reply:
This method is only called internally to get the actual path of an extra data file from its ID and name, but it cannot be called directly by a user.

Reply:
This method is only called internally to get the actual path of an extra data file from its ID and name, but it cannot be called directly by a user.